app/views/partials/_recovery_code.html.slim

```diff
@@ -0,0 +1,14 @@
+#recovery-code(class="col-12 border-box mt4 mb3 py2 px3 fs-20p sans-serif\
+                      border border-dashed border-red rounded-xl relative")
```

Comment:
If we wind up using this pattern anywhere else we can probably roll some of this up into a new class, but fine for now.